dotcom-rendering/src/palette.ts

```diff
@@ -1427,6 +1499,98 @@ const subNavBorder = ({ design, theme }: ArticleFormat): string => {
 	}
 };
 
+const pullQuoteTextLight = ({ design, theme }: ArticleFormat): string => {
+	switch (design) {
+		case ArticleDesign.Comment:
+		case ArticleDesign.Editorial:
+		case ArticleDesign.LiveBlog:
+		case ArticleDesign.DeadBlog:
+		case ArticleDesign.Analysis:
+		case ArticleDesign.Feature:
+		case ArticleDesign.Interview:
+		case ArticleDesign.Recipe:
+		case ArticleDesign.Review:
+			switch (theme) {
+				case Pillar.News:
+					return sourcePalette.news[200];
+				case Pillar.Opinion:
+					return sourcePalette.opinion[200];
+				case Pillar.Sport:
+					return sourcePalette.sport[200];
+				case Pillar.Culture:
+					return sourcePalette.culture[200];
+				case Pillar.Lifestyle:
+					return sourcePalette.lifestyle[200];
+				case ArticleSpecial.Labs:
+					return sourcePalette.labs[200];
+				case ArticleSpecial.SpecialReport:
+					return sourcePalette.specialReport[200];
+				case ArticleSpecial.SpecialReportAlt:
+					return sourcePalette.specialReportAlt[200];
+			}
+		default:
+			return sourcePalette.neutral[7];
+	}
+};
+
+const pullQuoteTextDark = ({ design, theme }: ArticleFormat): string => {
+	switch (design) {
+		case ArticleDesign.Comment:
+		case ArticleDesign.Editorial:
+		case ArticleDesign.LiveBlog:
+		case ArticleDesign.DeadBlog:
+		case ArticleDesign.Analysis:
+		case ArticleDesign.Feature:
+		case ArticleDesign.Interview:
+		case ArticleDesign.Recipe:
+		case ArticleDesign.Review:
+			switch (theme) {
+				case Pillar.News:
+					return sourcePalette.news[500];
+				case Pillar.Opinion:
+					return sourcePalette.opinion[500];
+				case Pillar.Sport:
+					return sourcePalette.sport[500];
+				case Pillar.Culture:
+					return sourcePalette.culture[500];
+				case Pillar.Lifestyle:
+					return sourcePalette.lifestyle[500];
+				case ArticleSpecial.Labs:
+					return sourcePalette.labs[400];
+				case ArticleSpecial.SpecialReport:
+					return sourcePalette.specialReport[500];
+				case ArticleSpecial.SpecialReportAlt:
+					return sourcePalette.specialReportAlt[700];
+			}
+		default:
+			return sourcePalette.neutral[97];
+	}
+};
+
+const pullQuoteBackgroundLight = (format: ArticleFormat): string => {
+	const articleBackground = decidePalette(format).background.article;
```

Comment:
Aware that we're moving away from `decidePalette`. My intention is to move the code inside `palette` in a follow-up refactoring PR as this one is quite big but happy to push the change here if people think it's best!

Reply:
Good catch - I'll make this change!